### ISSUE TITLE
Fail using SpotBugs 4.1.0

### ISSUE THREAD

**Issue by jorsol** [spotbugs/spotbugs-maven-plugin#243]
Running spotbugs-maven-plugin:4.0.4 and SpotBugs 4.1.0 fails with:

```
[INFO] --- spotbugs-maven-plugin:4.0.4:spotbugs (spotbugs) @ cm-api ---
Downloading from central: https://repo.maven.apache.org/maven2/com/github/spotbugs/spotbugs/4.1.0/spotbugs-4.1.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/github/spotbugs/spotbugs/4.1.0/spotbugs-4.1.0.pom (6.0 kB at 22 kB/s)
[WARNING] The POM for com.github.spotbugs:spotbugs:jar:4.1.0 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
Downloading from central: https://repo.maven.apache.org/maven2/com/github/spotbugs/spotbugs/4.1.0/spotbugs-4.1.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/github/spotbugs/spotbugs/4.1.0/spotbugs-4.1.0.jar (3.4 MB at 9.7 MB/s)
[INFO] Fork Value is true
     [java] Error: Unable to initialize main class edu.umd.cs.findbugs.FindBugs2
     [java] Caused by: java.lang.NoClassDefFoundError: org/dom4j/DocumentException
```

**Comment by hazendaz** [comment]
The pom mentions that it is invalid.  Try clearing your .m2 cache and retrying.

**Comment by pzygielo** [comment]
@jorsol Skip upgrading sb to 4.1.0 and use just released 4.1.1 which fixes this.

**Comment by hazendaz** [comment]
@pzygielo Thanks for the info!
